tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AdoNet/DbScopeFactory.cs

```diff
@@ -68,12 +68,15 @@ private static Scope CreateDbCommandScope(Tracer tracer, IDbCommand command, Int
                 if (tracer.Settings.DbmPropagationMode != DbmPropagationLevel.Disabled
                     && command.CommandType != CommandType.StoredProcedure)
                 {
-                    var propagatedCommand = DatabaseMonitoringPropagator.PropagateSpanData(tracer.Settings.DbmPropagationMode, tracer.DefaultServiceName, scope.Span.Context, integrationId);
+                    var propagatedCommand = DatabaseMonitoringPropagator.PropagateSpanData(tracer.Settings.DbmPropagationMode, tracer.DefaultServiceName, scope.Span.Context, integrationId, out var traceParentInjected);
 
                     if (propagatedCommand != string.Empty)
                     {
                         command.CommandText = $"{propagatedCommand} {commandText}";
-                        tags.DbmDataPropagated = "true";
+                        if (traceParentInjected)
+                        {
+                            tags.DbmDataPropagated = "true";
```

Comment:
It might be worth renaming this property to `DbmTraceInjected` to better align with the string value and our new understanding of the tag usage

Reply:
Makes sense! I updated in on: [0a4495a](https://github.com/DataDog/dd-trace-dotnet/pull/4483/commits/0a4495a620b09419f7688880a3374e34394063cf)